blueocean-dashboard/src/main/js/LoadingIndicator.js

```diff
@@ -44,7 +44,10 @@ export default {
     },
 
     hide() {
-        loadingCount--;
```

Comment:
Right, shouldn't be an issue, since people wouldn't go directly to it typically. Although, this seems like a fine change to make sure.